tests/infrared/16.1/stf-connectors.yaml.template

```diff
@@ -0,0 +1,105 @@
+        OS::TripleO::Services::AodhNotifier: /usr/share/openstack-tripleo-heat-templates/deployment/aodh/aodh-notifier-container-puppet.yaml
+        OS::TripleO::Services::AodhListener: /usr/share/openstack-tripleo-heat-templates/deployment/aodh/aodh-listener-container-puppet.yaml
+        OS::TripleO::Services::PankoApi: /usr/share/openstack-tripleo-heat-templates/deployment/deprecated/panko/panko-api-container-puppet.yaml
+
+    parameter_defaults:
+        CeilometerEnableGnocchi: true
+        GnocchiArchivePolicy: 'high'
+        GnocchiBackend: 'rbd'
+        GnocchiRbdPoolName: 'metrics'
+
+        EventPipelinePublishers: ['gnocchi://?filter_project=service&archive_policy=low']
+        MetricPipelinePublishers: ['gnocchi://?filter_project=service&archive_policy=low']
+        CeilometerQdrPublishEvents: true
+        CeilometerQdrPublishMetrics: true
+        CeilometerEnablePanko: false
+        ManagePolling: true
+        ManagePipeline: true
+        NotificationDriver: 'messagingv2'
+        CeilometerQdrEventsConfig:
+            driver: amqp
+            topic: event
+        CeilometerQdrMetricsConfig:
+            driver: amqp
+            topic: metering
+
+        CollectdEnableSensubility: true
+        CollectdSensubilityLogLevel: DEBUG
+        CollectdAmqpInstances:
+            notify:
+                format: JSON
+                notify: true
+                presettle: false
+            telemetry:
+                format: JSON
+                presettle: false
+        CollectdAmqpInterval: 5
+        CollectdConnectionType: amqp1
+        CollectdDefaultPollingInterval: 5
+
+        EnableSTF: true
+
+        MetricsQdrAddresses:
+        -   distribution: multicast
+            prefix: collectd
+        -   distribution: multicast
+            prefix: anycast/ceilometer
+
+        MetricsQdrSSLProfiles:
+        -   name: sslProfile
+
+        CephStorageExtraConfig:
+            tripleo::profile::base::metrics::collectd::amqp_host: "%{hiera('storage')}"
+            tripleo::profile::base::metrics::qdr::listener_addr: "%{hiera('storage')}"
+
+        ExtraConfig:
+            ceilometer::agent::polling::polling_interval: 5
+            collectd::plugin::cpu::reportbycpu: true
+            collectd::plugin::vmem::verbose: true
+            collectd::plugin::amqp1::send_queue_limit: 50
+            collectd::plugin::virt::extra_stats: "cpu_util disk vcpu"
+            collectd::plugin::ceph::daemons:
+               - ceph-osd.0
+               - ceph-osd.1
+               - ceph-osd.2
+               - ceph-osd.3
+               - ceph-osd.4
+               - ceph-osd.5
+               - ceph-osd.6
+               - ceph-osd.7
+               - ceph-osd.8
+               - ceph-osd.9
+               - ceph-osd.10
+               - ceph-osd.11
+               - ceph-osd.12
+               - ceph-osd.13
+               - ceph-osd.14
+               - ceph-osd.15
+
+        CollectdExtraPlugins:
+        - vmem
```

Comment:
Do we need the `- ceph` plugin added to this list?

Reply:
nope, it is installed by default on ceph nodes, but silent, if you don't add daemon configurations.

Reply:
What daemon configurations would need to be added? I had a ceph cluster up and running with the 16/ version of the script, but I don't remember seeing ceph plugin metrics.

Reply:
The change I did here is enough. I hope to have this kind of enumeration would not be necessary once we switch to ansible.